ProfileStore.luau

```diff
@@ -1250,16 +1279,18 @@ function ProfileStore.New(store_name, template)
 
 	template = template or {}
 
+	Constants = table.freeze(Constants)
```

Comment:
The table gets frozen when `table.freeze` is called. So, we don't need to set it equal to itself.
This line could be just this instead:
```luau
table.freeze(Constants)
```